mediation-old/src/main/java/com/wso2telco/dep/mediator/ResponseHandler.java

```diff
@@ -194,13 +175,7 @@ public String makePaymentResponse(String jsonBody, String clientCorrelator, Stri
 
 	 public String makePaymentResponseContext(MessageContext context, String jsonBody, String clientCorrelator, String requestResourceURL, String requestid) throws JSONException {
 
-         String jsonPayload = null;
-         FileReader fileReader = new FileReader();
-         String file = CarbonUtils.getCarbonConfigDirPath() + File.separator
-                       + FileNames.MEDIATOR_CONF_FILE.getFileName();
- 		Map<String, String> mediatorConfMap = fileReader.readPropertyFile(file);
-
-         Gson gson = new GsonBuilder().serializeNulls().create();
+		 Gson gson = new GsonBuilder().serializeNulls().create();
```

Comment:
Formatting

Reply:
Please check the source code. this might be a presentaion problem in git hub. 
Correctly formatted in the source.